jasmine/spec/inverted-index-spec.js

```diff
@@ -53,25 +49,23 @@ describe('testObj', function() {
 
       it('ensures the correct results when a single term is searched',
         function() {
-
-          expect(testObj.searchIndex('alice')).toEqual([0]);
-          expect(testObj.searchIndex('alliance')).toEqual([1]);
-          expect(testObj.searchIndex('joy')).toEqual([-1]);
+          expect(testObj.searchIndex('alice')).toEqual([[0]]);
+          expect(testObj.searchIndex('alliance')).toEqual([[1]]);
+          expect(testObj.searchIndex('joy')).toEqual([[-1]]);
         });
 
       it('ensures the correct results when multiple terms are searched',
         function() {
           // array
-          expect(testObj.searchIndex(['alice', 'lord'])).toEqual([0, 1]);
+          expect(testObj.searchIndex(['alice', 'lord'])).toEqual([[0], [1]]);
```

Comment:
**Code Quality - Nitpicks** 
Changed search function to return an array of arrays.